django/db/migrations/loader.py

```diff
@@ -64,6 +64,10 @@ def load_disk(self):
         self.unmigrated_apps = set()
         self.migrated_apps = set()
         for app_config in apps.get_app_configs():
+            if (self.connection is not None and
```

Comment:
I'd rather see that change in `migrations_module()` so Django doesn't suddenly pick up a module somewhere else but just doesn't know about migrations for any apps.